packages/services/src/contents/index.ts

```diff
@@ -1576,3 +1625,236 @@ namespace Private {
     return extension;
   }
 }
+
+class ContentProviderRegistry implements IContentProviderRegistry {
```

Comment:
Maybe we can add some docstrings to this class and methods, for consistency with the rest of the code base?

Reply:
Done in 6f337f0eadb2ff82b100686ea413350bef2ada29.